advisories/1.1.3/BRSA-bfjfsvvpma5s.toml

```
@@ -0,0 +1,17 @@
+[advisory]
+id = "BRSA-bfjfsvvpma5s"
+title = "kernel CVE-2024-49994"
+cve = "CVE-2024-49994"
+severity = "moderate"
+description = "In the Linux kernel, the following vulnerability has been resolved: block: fix integer overflow in BLKSECDISCARD"
+
+[[advisory.products]]
+package-name = "kernel-6.1"
+patched-version = "6.1.128"
+patched-epoch = "0"
+
+[updateinfo]
+author = "mharrimn"
+issue-date = 2025-02-24T21:37:50Z
```

Can you update the `issue-date` for these? It looks like these are using the `created_at` time rather than `published_at`

```
$ curl -Ls https://api.github.com/repos/bottlerocket-os/bottlerocket-kernel-kit/releases/latest | jq -r .published_at
2025-02-25T18:22:32Z
```

```suggestion
issue-date = 2025-02-25T18:22:32Z
```

---

Done.